src/application/CLI/CommandLineInterface.java

```diff
@@ -454,4 +500,37 @@ public static void editClassMenuHelp(){
             Option 6 - Return to Diagram Menu: returns the user to the diagram menu holding the class
                 """);
     }
+
+    public static boolean AutoCompleteControl(final String command, boolean shouldTerminate) {
```

Comment:
Same naming issue as the other method. Since this isn't related to auto-complete the method name is misleading